src/main/resources/db/migration/V1_9__add_vedtak_ferdig_published.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE VEDTAK ADD COLUMN ferdigbehandlet_published_at TIMESTAMPTZ;
```

Comment:
Dette fungerer greit sånn det er nå, men hvis det noen gang blir flere statuser enn de to vi har nå, så må vi fortsette å utvide med nye `published_at`-felter.

Reply:
Kan vurdere å ha dette i en egen vedtak-statusendring-tabell som vi inserter til når vedtak fattes og ferdigbehandles.

Reply:
Ja, enig i at dette ikke skalerer noe særlig. Kanskje jeg skal begynne med en egen PR for å tilpasse modellen først?